tests/telemetry/test_telemetry.py

```diff
@@ -19,14 +21,28 @@ def get_list_stdout(cmd_out):
         out_list.append(result)
     return out_list
 
+def restart_telemetry(duthost):
+    """ Restart telemetry process
+    """
+    duthost.shell('sudo config feature telemetry disabled', module_ignore_errors=False)['stdout_lines']
+    status_out = duthost.shell('show features | grep telemetry', module_ignore_errors=False)['stdout_lines']
+    status_value = get_list_stdout(status_out)
+    status_value_list = status_value[0].split()
+    status_value = status_value_list[1]
+    if str(status_value) == "disabled":
+        logger.info(' telemetry is disabled. enabling it back...')
```

Comment:
Remove leading space from message, and suggest to remove "back", i.e.,

```
logger.info(' telemetry service is disabled. Enabling it ...')
```

Reply:
removed logline. no longer needed.